.rust_alpha/kms/CreateKey/src/main.rs

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+use std::process;
+
+use kms::{Client, Config, Region};
+
+use aws_types::region::ProvideRegion;
+
+use structopt::StructOpt;
+use tracing_subscriber::fmt::format::FmtSpan;
+use tracing_subscriber::fmt::SubscriberBuilder;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// The region. Overrides environment variable AWS_DEFAULT_REGION.
+    #[structopt(short, long)]
+    default_region: Option<String>,
+
+    /// Activate verbose mode    
+    #[structopt(short, long)]
+    verbose: bool,
+}
+/// Creates an AWS KMS key.
```

Comment:
AWS KMS -> AWS Key Management Service (AWS KMS)
(first use in file)

Reply:
Fixed